ui/app/styles/components/env-banner.scss

```diff
@@ -4,7 +4,7 @@
  */
 
 .env-banner {
```

Comment:
Styling updates so it looks good in the footer:
<img width="1252" alt="Dev mode badge" src="https://github.com/hashicorp/vault/assets/82459713/3153f296-f770-40c4-a0dd-b55746cbb65f">